src/platform/site-wide/loading-button/LoadingButton.jsx

```diff
@@ -20,20 +21,21 @@ export default function LoadingButton({
 
   // Switch to va-button-icon once the loading icon is added
   return (
-    <button
-      className="usa-button"
+    <va-button
       {...props}
+      class={className}
       disabled={isLoading || disabled}
       onClick={onClick}
       aria-live="polite"
-    >
-      {contents}
-    </button>
+      text={contents}
+      role="button"
+    />
```

Comment:
I think this will break the loading animated icon

Reply:
Because the va-button will only accept text as the prop for `text` so passing in the div / children will be the issue

Reply:
@adamwhitlock1 ah that makes sense. I was having an issue with the button. I'm going to convert it to a va-button-icon

Reply:
not having a decent loading button in the design system components is really the root of the issue. We opened a ticket to propose adding one, but its in the backlog.

From what I can tell, there just needs to be a regular button here for the time being until something else becomes a viable option. Always open to suggestions otherwise though.

Reply:
Reverted it back to a button. I made the loader look a lot better too